test/tests_for_auth/ee2_admin_mode_test.py

```diff
@@ -104,13 +108,22 @@ def get_runner_with_condor(self) -> SDKMethodRunner:
 
     # TODO How do you test ADMIN_MODE without increasing too much coverage
 
+    def get_mocks(self) -> (UserClientSet, WorkspaceAuth):
+        # add workspace client mock if needed
+        ucs = create_autospec(UserClientSet, instance=True, spec_set=True)
+        wsa = create_autospec(WorkspaceAuth, instance=True, spec_set=True)
+        ucs.workspace_auth.return_value = wsa
+        return ucs, wsa
+
     @patch.object(Catalog, "get_module_version", return_value="module.version")
-    @patch.object(WorkspaceAuth, "can_write", return_value=True)
     @patch.object(AdminAuthUtil, "_fetch_user_roles")
-    def test_regular_user(self, aau, workspace, catalog):
+    def test_regular_user(self, aau, catalog):
         # Regular User
         lowly_user = "Access Denied: You are not an administrator"
-        runner = self.getRunner()
+        ucs, wsa = self.get_mocks()
```

Comment:
It would help with readability to write out 
`user_client_set` and `ws_auth` instead of these 3 letter variables names

Reply:
done